packages/design-system-docs/src/styles/components/_ReactPropDocs.scss

```diff
@@ -0,0 +1,54 @@
+// Responsive docs table
```

Comment:
I'm thinking we should probably use display and responsive utility classes in the HTML, Then I don't think you'd need to use the custom SCSS because you could use utility classes. 

- https://design.cms.gov/utilities/display-visibility/#utilities.display-visibility.visibility
- https://design.cms.gov/utilities/display-visibility/#utilities.display-visibility.responsive

Reply:
There isn't a responsive utility for `ds-u-visibility--screen-reader`, so we would need custom SCSS anyway I believe. Maybe we could use the class via an extend though: 
`@extend .ds-u-visibility--screen-reader`

Reply:
The `@extend .ds-u-visibility--screen-reader` would be ideal for the table header on small screens but extending selector only works within the same media context. A work around can be to explicitly create a mixin for the ds-u-visibility--screen-reader in the utility class and then we can import the mixin in the media query. For this ticket, i thought it best not to change the utility class.

Reply:
ah thats right, you mentioned you cant extend in a media. i think what you have here looks good!